clang/docs/analyzer/checkers.rst

```diff
@@ -2097,6 +2097,18 @@ This checker is a part of ``core.StackAddressEscape``, but is temporarily disabl
 
 .. _alpha-core-TestAfterDivZero:
 
+alpha.core.StdVariant (C++)
+"""""""""""""""""""""""""""
+Check if the active value accessed in an ``std::variant`` instance.
+.. code-block:: cpp
+
+ void test() {
+   std::variant <int, char> v = 25;
```

Comment:
```suggestion
   std::variant<int, char> v = 25;
```

Reply:
I don't think this comment was resolved.